pkg/lightning/restore/check_info.go

```diff
@@ -238,7 +259,7 @@ func (rc *Controller) CheckpointIsValid(ctx context.Context, tableInfo *md.MDTab
 	tableCheckPoint, err := rc.checkpointsDB.Get(ctx, uniqueName)
 	if err != nil {
 		// there is no checkpoint
-		log.Debug("no checkpoint detected", zap.String("table", uniqueName))
+		log.L().Debug("no checkpoint detected", zap.String("table", uniqueName))
```

Comment:
???

Reply:
The origin code use another `log` dependency ,which is usually used by `BR` rather than lightning